### DISCUSSION TITLE
Java client for lastest seaweedfs

### DISCUSSION THREAD

**Discussion by samshihongxin** [seaweedfs/seaweedfs#2072] [Q&A]
Hi, can u share java client for lastest seaweedfs?

**Comment by chrislusf** [comment]
Added instructions here https://github.com/chrislusf/seaweedfs/wiki/SeaweedFS-Java-Client#build-java-client-jar

**Reply by samshihongxin** [reply]
thx

**Reply by samshihongxin** [reply]
Hi, is there java client that supports filer cluster?

**Reply by chrislusf** [reply]
Not yet.
